content/docs/fibers/error_handling.mdz

```diff
@@ -43,3 +49,16 @@ common case.
    (print "will never get here"))
  ([err] (print "caught error: " err)))
 ```
+
+### @code`protect`
+
+Janet also provides the @code`protect` macro for a slightly different
+flavor of error-handling, converting caught errors into further
```

Comment:
I would go with something like "returning the tuple of two members, [true result] or [false error]" but in your fine language tho. And the nice example I see here :).